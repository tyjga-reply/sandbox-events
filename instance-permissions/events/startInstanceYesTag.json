{
    "version": "0",
    "id": "dcac2ebb-db88-96df-9b52-b4e8c24500fc",
    "detail-type": "AWS API Call via CloudTrail",
    "source": "aws.ec2",
    "account": "211125361846",
    "time": "2024-12-17T10:37:23Z",
    "region": "us-east-1",
    "resources": [],
    "detail": {
        "eventVersion": "1.10",
        "userIdentity": {
            "type": "AssumedRole",
            "principalId": "AROATCKANLS3NIDXYUGTV:t.kapp@reply.de",
            "arn": "arn:aws:sts::211125361846:assumed-role/AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4/t.kapp@reply.de",
            "accountId": "211125361846",
            "accessKeyId": "ASIATCKANLS3CRHZXG7X",
            "sessionContext": {
                "sessionIssuer": {
                    "type": "Role",
                    "principalId": "AROATCKANLS3NIDXYUGTV",
                    "arn": "arn:aws:iam::211125361846:role/aws-reserved/sso.amazonaws.com/eu-central-1/AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4",
                    "accountId": "211125361846",
                    "userName": "AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4"
                },
                "attributes": {
                    "creationDate": "2024-12-17T08:29:50Z",
                    "mfaAuthenticated": "false"
                }
            }
        },
        "eventTime": "2024-12-17T10:37:23Z",
        "eventSource": "ec2.amazonaws.com",
        "eventName": "StartInstances",
        "awsRegion": "us-east-1",
        "sourceIPAddress": "91.102.9.132",
        "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.1.1 Safari/605.1.15",
        "requestParameters": {
            "instancesSet": {
                "items": [
                    {
                        "instanceId": "i-0988c71975a66d495"
                    }
                ]
            }
        },
        "responseElements": {
            "requestId": "c582e748-9be5-40d7-a0a8-a6833bca3aae",
            "instancesSet": {
                "items": [
                    {
                        "instanceId": "i-0988c71975a66d495",
                        "currentState": {
                            "code": 0,
                            "name": "pending"
                        },
                        "previousState": {
                            "code": 80,
                            "name": "stopped"
                        }
                    }
                ]
            }
        },
        "requestID": "c582e748-9be5-40d7-a0a8-a6833bca3aae",
        "eventID": "78068511-43b7-4d45-ac4a-7e60070b0f1d",
        "readOnly": false,
        "eventType": "AwsApiCall",
        "managementEvent": true,
        "recipientAccountId": "211125361846",
        "eventCategory": "Management",
        "tlsDetails": {
            "tlsVersion": "TLSv1.3",
            "cipherSuite": "TLS_AES_128_GCM_SHA256",
            "clientProvidedHostHeader": "ec2.us-east-1.amazonaws.com"
        },
        "sessionCredentialFromConsole": "true"
    }
}