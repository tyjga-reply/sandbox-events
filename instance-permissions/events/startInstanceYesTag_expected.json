{"statusCode": 200, "body": "{\"message\": \"Resource ID extracted successfully!\", \"eventName\": \"StartInstances\", \"resourceId\": \"i-0988c71975a66d495\", \"valid\": true, \"ipInfo\": {\"PrivateIpAddress\": \"172.31.89.183\", \"PublicIpAddress\": \"N/A\"}}"}